crates/volta-migrate/src/v1.rs

```diff
@@ -87,17 +87,18 @@ impl TryFrom<V0> for V1 {
 
             debug!("Removing old Volta binaries");
             let old_volta_bin = new_home.root().join("volta");
-            if old_volta_bin.exists() {
-                remove_file(&old_volta_bin).with_context(|| ErrorKind::DeleteFileError {
+
+            remove_file(&old_volta_bin)
+                .or_else(ok_if_not_found)
+                .with_context(|| ErrorKind::DeleteFileError {
                     file: old_volta_bin,
                 })?;
-            }
 
             let old_shim_bin = new_home.root().join("shim");
-            if old_shim_bin.exists() {
-                remove_file(&old_shim_bin)
-                    .with_context(|| ErrorKind::DeleteFileError { file: old_shim_bin })?;
-            }
+
+            remove_file(&old_shim_bin)
+                .or_else(ok_if_not_found)
```

Comment:
Same here about using `remove_file_if_exists` instead of `remove_file().or_else()`